ginga/misc/plugins/SaveImage.py

```diff
@@ -0,0 +1,493 @@
+        b.channel_name.add_callback('activated', self.select_channel_cb)
+
+        b.outdir.set_text(self.outdir)
+        b.outdir.set_tooltip('Output directory')
+        b.outdir.add_callback('activated', lambda w: self.set_outdir())
+
+        b.browse.set_tooltip('Browse for output directory')
+        b.browse.add_callback('activated', lambda w: self.browse_outdir())
+
+        b.suffix.set_text(self.suffix)
+        b.suffix.set_tooltip('Suffix to append to filename')
+        b.suffix.add_callback('activated', lambda w: self.set_suffix())
+
+        mod_only = self.settings.get('modified_only', True)
+        b.modified_only.set_state(mod_only)
+        b.modified_only.add_callback('activated',
+                                            lambda *args: self.redo())
+        b.modified_only.set_tooltip("Show only locally modified images")
+
+        container.add_widget(w, stretch=0)
+
+        self.treeview = Widgets.TreeView(auto_expand=True,
+                                         sortable=True,
+                                         selection='multiple',
+                                         use_alt_row_color=True)
+        self.treeview.setup_table(self.columns, 1, 'IMAGE')
+        self.treeview.add_callback('selected', self.toggle_save_cb)
+        container.add_widget(self.treeview, stretch=1)
+
+        captions = (('Status', 'llabel'), )
+        w, b = Widgets.build_info(captions, orientation=orientation)
+        self.w.update(b)
+        b.status.set_text('')
+        b.status.set_tooltip('Status message')
+        container.add_widget(w, stretch=0)
+
+        btns = Widgets.HBox()
+        btns.set_border_width(4)
+        btns.set_spacing(3)
+
+        btn = Widgets.Button('Save')
+        btn.set_tooltip('Save selected image(s)')
+        btn.add_callback('activated', lambda w: self.save_images())
+        btn.set_enabled(False)
+        btns.add_widget(btn, stretch=0)
+        self.w.save = btn
+
+        btn = Widgets.Button('Close')
+        btn.add_callback('activated', lambda w: self.close())
+        btns.add_widget(btn, stretch=0)
+        btns.add_widget(Widgets.Label(''), stretch=1)
+        container.add_widget(btns, stretch=0)
+
+        self.gui_up = True
+
+        # Initialize directory selection dialog
+        self.dirsel = DirectorySelection(self.fv.w.root.get_widget())
+
+        # Generate initial listing
+        self.update_channels()
+
+    def instructions(self):
+        self.tw.set_text("""Enter output directory and suffix, if different than default. Left click to select image name to save. Multiple images can be selected using click with Shift or CTRL key. Click Save to save the selected image(s).
+
+Output image will have the filename of <inputname>_<suffix>.fits.""")
+
+    def redo(self, *args):
+        """Generate listing of images that user can save."""
+        if not self.gui_up:
+            return
+
+        #mod_only = self.settings.get('modified_only', True)
```

Comment:
The commented line can be deleted since we decided to keep the checkbox. Otherwise, it is just going to confuse people in the future.